fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -2498,7 +2498,6 @@ public void modifyTableAddOrDropInvertedIndices(Database db, OlapTable olapTable
         schemaChangeJob.setJobState(AlterJobV2.JobState.WAITING_TXN);
         this.addAlterJobV2(schemaChangeJob);
         LOG.debug("logAlterJob schemaChangeJob:{}", schemaChangeJob);
```

Comment:
you have remove the invoker of 'logAlterJob', may be the debug log also need to change.

Reply:
yes, the debug log also can be removed, thanks.